### DISCUSSION TITLE
Having problem loading widget because of crossorigin

### DISCUSSION THREAD

**Discussion by patt-rick** [chatwoot/chatwoot#6732] [Q&A]
i've added chatwoot to my react app. both the SDK and widget sends the request . but the response of the widget is blocked because of crossorigin . How do I fix this

**Comment by vishnu-narayanan** [comment]
What kind of deployment are you using? Have you tried setting up CORS on your server?

**Reply by patt-rick** [reply]
My project is a react application using sqlite for browser. so I set up some cors to enable the sqlite work in the browser. without it, the sqlite wouldn't work. I think this is what is causing my problem. If I comment that code out. it works fine but the whole application is based on the sqlite so I need it. Anyway I could go about it?

**Reply by vishnu-narayanan** [reply]
Could you share the CORS setup you need for SQLite to work?

**Reply by patt-rick** [reply]
`app.use(function (req, res, next) {
    res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
    res.setHeader("Cross-Origin-Opener-Policy", "same-origin");
    next();
  });`
Here's where i set it up
